### ISSUE TITLE
Add "Logs" expand button to the Dashboard 'Backend is not available.' view

### ISSUE THREAD

**Issue by ibuziuk** [eclipse/che#22423]
### Is your enhancement related to a problem? Please describe

Currently, if there is a 500 error from che-server dashboard shows Backend is not available view, but it is not clear at all what is the culprit. Browser, logs are not helpful:

![image](https://github.com/eclipse/che/assets/1461122/0dcda3e8-baed-4f64-a19e-504470ab964a)

### Describe the solution you'd like

![image](https://github.com/eclipse/che/assets/1461122/404606fe-d172-45ab-a675-9bdcdc83b839)

When clicking on the `Logs` the trace from 500 error is shown e.g.

```
Failure executing: POST at: https://172.30.0.1/api/v1/namespaces/osio-ci-devsandbox-periodic-stg-dev/secrets. Message: Forbidden!Configured service account doesn't have access. Service account may have been revoked. secrets \"user-profile\" is forbidden: exceeded quota: for-osio-ci-devsandbox-periodic-stg-secrets, requested: count/secrets=1, used: count/secrets=100, limited: count/secrets=100
```

### Additional context

Che-server should provide meaning error message when 500 happens which dashboard would display on UI

**Comment by vinokurig** [comment]
Removing the `area/che-server` label as the exception is already passed to the dashboard in the exception catch:
https://github.com/eclipse-che/che-dashboard/blob/8f95e78212740a7997034a620e7e3feaab80aedb/packages/dashboard-frontend/src/store/SanityCheck/index.ts#L110-L121

**Comment by ibuziuk** [comment]
@vinokurig could you please make sure that the fixes for this issue are backported to 3.9

**Comment by vinokurig** [comment]
@ibuziuk 
cherry picked to `7.74.x`, closing the issue.
